docs/topics/modules/using_stack_report.adoc

```diff
@@ -26,7 +26,7 @@ At the Deployment stage, {osio} Analytics analyzes your stack and provides a det
 
 .Prerequisites
 
-Ensure that you deploy your project and <<creating_a_new_pipeline, create a build pipeline>> for your project.
+Ensure that you deploy your project and <<creating_new_project-user-guide>> for your project.
```

Comment:
This introduced a mistake. Will fix in #227.

Reply:
I'm struggling to understand this change. The sentence would read:

_Ensure that you deploy your project and **Creating a new project** for your project._

Why was `creating_a_new_pipeline.adoc` deleted?

Reply:
OK, I get it now. This was not easy to decipher...